bin/next-dev

```diff
@@ -61,6 +61,16 @@ srv.start(argv.port)
   }
 })
 .catch((err) => {
-  console.error(err)
-  process.exit(1)
+  if (err.code === 'EADDRINUSE') {
+    let errorMessage = `Port ${argv.port} is already in use.`
+    try {
+      const appPackage = JSON.parse(readFileSync(join(dir, 'package.json'), 'utf8'))
+      const nextTuple = Object.entries(appPackage.scripts).find(i => i[1] === 'next')
```

Comment:
Also it's possible for users to give some additional stuff to the next command like: `next custom-dir-name`